openshiftcontrolplane/v1/types.go

```diff
@@ -239,6 +239,9 @@ type ImageImportControllerConfig struct {
 type BuildDefaultsConfig struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// proxyConfig is the location of the default Proxy Configuration
+	ProxyConfig configv1.ProxySpec `json:"proxyConfig,omitempty"`
```

Comment:
/s/ProxyConfig/DefaultProxy/

Also use pointer if we decide to go this route - we'll need to distinguish if this was set at all vs. individual values set to empty.

Reply:
Aren't we switching this to go through the buildDefaults? If so, this seems like the place to put it.